packages/origin-backend-app/src/mail/mail.service.ts

```diff
@@ -19,30 +20,23 @@ interface IMandrillMailSendStatus {
 export class MailService {
     private readonly logger = new Logger(MailService.name);
 
-    constructor(private readonly mailerService: MailerService) {}
+    constructor(
+        private readonly mailerService: MailerService,
+        private readonly configService: ConfigService
+    ) {}
 
     async send(sendMailOptions: ISendMailOptions) {
         try {
-            const result: IMandrillMailSendStatus = await this.mailerService.sendMail(
-                sendMailOptions
-            );
-
-            if (!result.messageId) {
-                return true;
-            }
-
-            const allSucceeded = result.accepted.every((e) => e.status === 'sent');
-
-            if (allSucceeded) {
-                this.logger.log(`Sent email with id: ${result.messageId}. `);
-                this.logger.log(result);
```

Comment:
This removes the log on the sent email. We probably want to keep a log of sent emails for debugging. Do we log this somewhere else?

Reply:
This new library does not have valid mandrill message status, I've added it here
https://github.com/xr0master/mandrill-nodemailer-transport/pull/5
I'll restore this code after merge of PR above